wire-ios-request-strategy/Sources/Synchronization/Decoding/StoreUpdateEventTests.swift

```diff
@@ -271,6 +271,41 @@ class StoreUpdateEventTests: MessagingTestBase {
         }
     }
 
+    func test_EncryptAndCreate_DoesNotStoreDuplicateEvents() throws {
+        eventMOC.performAndWait {
+            // Given some events.
+            let conversation = self.createConversation(in: self.uiMOC)
+            let event1 = self.createNewConversationEvent(for: conversation)
```

Comment:
Probably good to insert another event with the same event to account for the possibility that we stored multiple events from the same envelope.

Reply:
done in another test